### DISCUSSION TITLE
Some UI design

### DISCUSSION THREAD

**Discussion by raresvanca** [slowscript/warpinator-android#5] [Show and tell]
I creted this ui design:

![ezgif-7-d060f2960206](https://user-images.githubusercontent.com/42469755/101682595-82301400-3a6c-11eb-89c5-f5d207335688.gif)

It's just an idea and not exactly what I have in mind. Is more of a concept. I just want your opinions.

**Comment by slowscript** [comment]
Sorry for the delay. I had a busy few days.

Most of it looks good, but when it comes to design, I would like to stay close to default android design if possible, or at least don't introduce things that require many changes to code and custom views.
Most of what you have done seems achievable with xml, so that shouldn't be a problem. I would however stay away from redefining toasts and having to slide away the card to accept a transfer doesn't look very intuitive to me. Also I'm curious how it would look in light mode.

Just my 2 cents.

**Comment by raresvanca** [comment]
![image](https://user-images.githubusercontent.com/42469755/101980488-65632e80-3c6e-11eb-9e95-8e3c391356c3.png)![image](https://user-images.githubusercontent.com/42469755/101980495-7449e100-3c6e-11eb-9362-1c50d22746a1.png)![image](https://user-images.githubusercontent.com/42469755/101980503-8461c080-3c6e-11eb-8938-2a835ad4a5a6.png)![image](https://user-images.githubusercontent.com/42469755/101980522-ab1ff700-3c6e-11eb-8a81-3ddad9a84878.png)

My suggestion for the light mode.

About toasts you are right. I'm gonna use the default toasts.
